.github/workflows/release.yml

```diff
@@ -0,0 +1,97 @@
+name: Docker RELEASE
+
+on:
+    schedule:
+        -   cron: '30 2 * * *'
+    push:
+        tags:
+            - 'v*.*'
+env:
+    IMAGE_NAME: pimcore/pimcore
+
+jobs:
+    build-php:
+        name: "Build PHP images"
+        runs-on: ubuntu-latest
+        if: github.repository == 'pimcore/docker'
+        strategy:
+            matrix:
+                php: [ '8.0', '8.1' ]
+                distro: [ bullseye ]
+                target: [ fpm, debug, supervisord ]
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            -   name: Login to DockerHub Registry
+                run: echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
+
+            -   name: Cache Docker layers
+                uses: actions/cache@v2
+                with:
+                    path: /tmp/.buildx-cache
+                    key: ${{ runner.os }}-buildx-${{ matrix.php }}-${{ matrix.distro }}-${{ matrix.target }}
+                    restore-keys: |
+                        ${{ runner.os }}-buildx-${{ matrix.php }}-${{ matrix.distro }}-${{ matrix.target }}-
+
+            -   name: Set up QEMU
+                uses: docker/setup-qemu-action@v2
+                with:
+                    platforms: 'amd64,arm64'
+
+            -   name: Set up Buildx
+                uses: docker/setup-buildx-action@v2
+
+            -   name: Set Release Name
+                id: vars
+                run: |
+                    DOCKER_PLATFORMS=linux/amd64,linux/arm64
+                    PHP_VERSION=${{ matrix.php }}
+                    DEBIAN_VERSION="${{ matrix.distro }}
+                    TARGET=pimcore_php_${{ matrix.target }}
+                    VERSION="edge"
+                    BASE_TAG="php${{ matrix.php }}-${{ matrix.distro }}-${{ matrix.target }}"
+                    
+                    if [ "${{ github.event_name }}" = "schedule" ]; then
+                        VERSION="latest"
+                    fi
+                    
+                    if [[ $GITHUB_REF == refs/tags/* ]]; then
+                        VERSION=${GITHUB_REF#refs/tags/v}
+                    fi
+
+                    TAG="${BASE_TAG}-${VERSION}"
+                    TAGS="--tag ${IMAGE_NAME}:${TAG}"
+                    
+                    # Tag latest with Version build too
+                    if [[ $VERSION =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
+                        TAGS="$TAGS --tag ${IMAGE_NAME}:${BASE_TAG}-latest"
+                    fi
+
+                    echo ::set-output name=docker_image::${DOCKER_IMAGE}
```

Comment:
I'm missing the `-dev` tag here. 
Should we also include an alias for `cli` for backward compatibility and also one tag without the distro, e.g. `pimcore/pimcore:PHP8.1-fpm`

And maybe we should also include already and alias for the patch version of PHP, so e.g. `pimcore/pimcore:PHP8.1.1-fpm-v2.0.0` as discussed here https://github.com/pimcore/docker/issues/34#issuecomment-1202307256

Reply:
What is dev `-dev`? Latest is build on changes and reguarly.

> And maybe we should also include already and alias for the patch version of PHP, so e.g. pimcore/pimcore:PHP8.1.1-fpm-v2.0.0 as discussed here https://github.com/pimcore/docker/issues/34#issuecomment-1202307256

That means we need to basically create matrix entries for all PHP Patch versions, who keeps track of that?

> Don't think we need to include fpm in the new tag schema anyway, but we should ensure backward compatibility with the current ones, right?

Well, basically how you like it better, it is easier for me to add it cause the matrix entries is called like that :D

Reply:
`-dev` yep, so basically daily builds based on `main`. 
PHP patch versions: no, I'd use something like that to get the version number for the current build: 
`docker run -it --rm php:8.1-fpm-bullseye php -v | grep -Po 'PHP \d.\d.\d'`

Reply:
@dpfaffenbauer any updates? 😊 Thanks!

Reply:
@brusch quite busy this week, will continue next week

Reply:
Thanks a lot 👍